platform-service/src/unstract/platform_service/controller/platform.py

```diff
@@ -20,12 +20,19 @@
 
 from unstract.flags.feature_flag import check_feature_flag_status
 
-be_db = peewee.PostgresqlDatabase(
+be_db = PooledPostgresqlDatabase(
     Env.PG_BE_DATABASE,
     user=Env.PG_BE_USERNAME,
     password=Env.PG_BE_PASSWORD,
     host=Env.PG_BE_HOST,
     port=Env.PG_BE_PORT,
+    max_connections=32,
+    #  Number of seconds to allow connections
+    # to be used. Same as gunicorn timeout
+    stale_timeout=900,
+    #  Number of seconds to block
+    # when pool is full. Set to 5 minutes.
+    timeout=300,
```

Comment:
NIT: @harini-venkataraman might be good to set and use these through envs

Reply:
Addressed this comment. `max_connections` value is dynamic and needs to be set based on load testing keeping the db engaged. Changing to env, to avoid code changes. Thanks for pointing it out @chandrasekharan-zipstack